src/renderer/reducers/notification.ts

```diff
@@ -0,0 +1,66 @@
+import {IStoreNotificationAction, IRemoveNotificationAction} from "../actions/notification";
+import {NotificationActionTypes} from "../constants/action-types";
+import {Action} from "redux";
+import {Level, Horizontal, Vertical} from "../components/Notification/NotificationEnums";
+
+export interface INotificationState {
+    isVisible: boolean,
+    title: string,
+    content: string,
+    level: Level,
+    horizontalPosition: Horizontal,
+    verticalPosition: Vertical
+}
+
+export interface INotificationStateObject {
+    stacked: Array<INotificationState>,
+    other: Array<INotificationState>
+}
+
+const initialState: INotificationStateObject = {
+    stacked: [],
+    other: []
+};
+
+export const notificationReducer = (
+    state = initialState, 
+    action: Action<NotificationActionTypes>): INotificationStateObject => {
+
+    // Copy of current notification state arrays
+    const newStackedArray = state.stacked.slice();
+    const newOtherArray = state.other.slice();
```

Comment:
This will always refresh UI generated by both arrays, you should create new array only if you are going to add something